getting-started-command-mode/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+# Quarkus Configuration file
```

Comment:
Can be removed.

Reply:
this is what is there in other quickstarts and makes it so users can just open the file and tweak without having to remember where it should be put....so I kept it. Objection?

Reply:
No, it's fine. Yes, the other quickstarts still have them.